builtin/logical/pki/backend_test.go

```diff
@@ -4189,9 +4189,10 @@ func TestBackend_RevokePlusTidy_Intermediate(t *testing.T) {
 
 	// Issue a tidy on /pki
 	_, err = client.Logical().Write("pki/tidy", map[string]interface{}{
-		"tidy_cert_store":    true,
-		"tidy_revoked_certs": true,
-		"safety_buffer":      "1s",
+		"tidy_cert_store":       true,
+		"tidy_revoked_certs":    true,
+		"safety_buffer":         "1s",
+		"revoked_safety_buffer": "1s",
```

Comment:
I would probably leave this test alone as well :-) We want to show that `revoked_safety_buffer` is fine to leave off the request, if you don't want to set it separately from `safety_buffer`.